src/app/core/app.component.ts

```diff
@@ -26,8 +27,8 @@ export class AppComponent implements OnInit, OnDestroy {
     this.localeService.currentLangError$,
   );
 
-  leftMenuDisplayed = true;
-  headersDisplayed = true;
+  leftMenuAndHeaders = true;
+  leftMenuAndHeadersSubscription? : Subscription;
```

Comment:
Is there a reason for keeping a extra copy of this state? 
You should just use the observable directly
```ts
leftMenuAndHeaders$ = this.layoutService.leftMenuAndHeadersDisplayed$;
```
and use it with the `async` pipe in the code.

Reply:
Thanks. I used `*ngrxLet` in the template as I saw another line doing this, and it avoids adding a lot of async pipes.